Makefile

```diff
@@ -0,0 +1,72 @@
+.DEFAULT_GOAL = help
+.PHONY: help install gitmodules build start stop wait-healthy sh exec logs watch run dev prod
+
+SHELL = /usr/bin/env bash
+
+ifneq (${TARGET}, prod)
```

Comment:
Not sure if the intent is to capture cases where target has not been set, or has been set to an invalid value. But for the former case you can replace this if construct with...
TARGET ?= dev
Which will set TARGET to 'dev' if target is not already defined.

Reply:
About the latter case (ideally would throw an error, suggestions welcome).

Having `TARGET=foo` would cause some commands to work fine, and others not. Forcing it to `dev` in that case makes the behaviour consistent.